azimuth_capi/config.py

```diff
@@ -145,11 +145,11 @@ class WebhookConfiguration(Section):
     #: Indicates whether kopf should manage the webhook configurations
     managed: bool = False
     #: The path to the TLS certificate to use
-    certfile: t.Optional[FilePath] = Field(None, validate_default = True)
+    certfile: t.Optional[FilePath] = Field(None, validate_default = False)
```

Comment:
Note: this is to allow the default settings to load, so we can just run the unit tests without needing to specify custom config.